LearningMissionLab/FinancialInfo.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace LearningMissionLab
+{
+    public class FinancialInfo
```

Comment:
Financialnfo should rely on FinancialEvent helper class. FinancialIfo is pretty much a list of independent financial events. Each financial event should have a type, start date, end date and the amount.